test/Confluent.SchemaRegistry.Serdes.IntegrationTests/Tests/ProduceConsumeGeneric.cs

```diff
@@ -52,48 +46,54 @@ public static void ProduceConsumeGeneric(string bootstrapServers, string schemaR
             );
 
             var config = new ProducerConfig { BootstrapServers = bootstrapServers };
-            var schemaRegistryConfig = new SchemaRegistryConfig { SchemaRegistryUrl = schemaRegistryServers };
+            var schemaRegistryConfig = new SchemaRegistryConfig
+            {
+                SchemaRegistryUrl = schemaRegistryServers,
+                // Test SchemaRegistryKeySubjectNameStrategy here,
+                // and SchemaRegistryValueSubjectNameStrategy in ProduceConsume.
+                SchemaRegistryKeySubjectNameStrategy = nameStrategy
```

Comment:
This reduces test coverage, would rather like to see an additional test that explicitly sets the parameter.

Reply:
it also extends test coverage (to previously applicable cases that weren't covered). i'm making an effort / payoff tradeoff decision here.